packages/voice/src/networking/DAVESession.ts

```diff
@@ -0,0 +1,243 @@
+import { Buffer } from 'node:buffer';
+import { EventEmitter } from 'node:events';
+import type { VoiceDavePrepareEpochData, VoiceDavePrepareTransitionData } from 'discord-api-types/voice/v8';
+import { DAVESessionError, DAVESessionErrorKind } from './DAVESessionError';
+
+const LIBRARY_NAME = '@snazzah/davey';
```

Comment:
I see this is used in 2 places:

- packages/voice/src/networking/DAVESession.ts (further down)
- packages/voice/src/util/generateDependencyReport.ts

Perhaps creating constants.ts in the util folder and putting this here would be worthwhile?